### PR TITLE
some minor changes/thoughts on intro

### PR DESCRIPTION
**Pull request by brettbj**
@cgreene - let me know what you think about the TODO- comments. Hoping to get to sections next couple of days.

### CODE REVIEW

File: sections/02_intro.md [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
+Biology and medicine are rapidly becoming particularly data-intensive with
 respect to both research and practice. A recent comparison of genomics with
 social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We expect that algorithms to automatically extract
-meaningful patterns and provide sufficient context to enable us to act will be
-required.
+also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
```
Comment (cgreene) [inline sections/02_intro.md]: `foresee` is a tall order. Maybe:

> We expect that achieving transformational advances will require algorithms that automatically extract meaningful patterns while providing actionable insights that enable us to better treat, study, and categorize disease, all within data constrained and privacy critical environments.

Comment (cgreene) [inline sections/02_intro.md]: (this incorporates suggestions made by @gwaygenomics on my draft and would supercede the changes we merged yesterday)

File: sections/02_intro.md [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 Computer scientists are now building many-layered neural networks from
-collections of millions of images. In a famous example, scientists from Google
+collections of millions of images. In an early famous example, scientists from Google
 demonstrated that a neural network could learn to identify cats simply by
 watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
 termed deep learning, seem like a solution to the challenge presented by the
 growth of data in biomedicine. Perhaps these algorithms could identify the
-biological "cats" hidden in our data - the patterns that exist but that we don't
-know to look for - and could act on them.
+biological "cats" hidden in our data - the patterns unknown to researcher - and could act on them.
```
Comment (cgreene) [inline sections/02_intro.md]: suspect this merge will return `biological` - can you remove it at next commit?

Comment (cgreene) [inline sections/02_intro.md]: It's also a bit unclear whether the algorithms or researcher are acting. It was unclear before so feel free to chose. And there's some grammar issues with the bit between the -s

File: sections/02_intro.md [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
 
-Deep learning has transformed image analysis, but researchers' initial forays
-into the use of these techniques in biomedicine have been relatively limited.
+
+Deep learning has since transformed image analysis achieving human-level results in several tasks [@doi:10.1007/s11263-015-0816-y]. In addition, deep learning has been revolutionary in speech recognition and synthesis [arXiv:1412.5567v2,   arXiv:1609.08144], game-playing and reinforcement learning [@doi:10.1038/nature16961, doi:10.1038/nature14236], and natural language processing [   arXiv:1310.4546, arXiv:1609.08144]. In contrast, researchers' initial forays into the use of these techniques in biomedicine have been relatively limited.
 There are certainly numerous promising examples, but we have not yet seen the
-massive convergence on these approaches that occurred in the field of image
-analysis. In this review, we discuss whether this is simply a matter of time or
+massive convergence on these approaches that occurred in other fields. In this review, we examine whether this is simply a matter of time or
```
Comment (cgreene) [inline sections/02_intro.md]: I know `massive` was in there before, but `convergence` is probably sufficient. I suggest removal.

File: sections/02_intro.md [hunk 4]
```diff
@@ -38,16 +34,16 @@ inputs is usually multiplied by some weight and combined and summarized by the
 activation function. The value of the activation function is then multiplied by
 another set of weights to produce the output `TODO: we probably need a figure
 here - I see no way that we don't include this type of description in our paper,
-despite the fact that it's been done tons of times before.` These neural
+despite the fact that it's been done tons of times before. - I'm really partial to the nature bengio revew's presentation - really hitting on the idea of learning a feature to make a non-linear problem linear` These neural
```
Comment (cgreene) [inline sections/02_intro.md]: Do you mean Figure 1A of the Perspective article they had in Nature (LeCun, Bengio and Hinton)?

File: sections/02_intro.md [hunk 5]
```diff
@@ -38,16 +34,16 @@ inputs is usually multiplied by some weight and combined and summarized by the
...
 some specific input.
 
 Neural networks can also be stacked. The outputs from one can be used as inputs
 to another. This process produces a stacked or multi-layer neural network. The
 multi-layer neural network techniques that underlie deep learning have a long
-history. Multi-layer methods have been discussed in the literature for more than
+history, having been discussed in the literature for more than
 five decades [@doi:10.1103/RevModPhys.34.135]. Given this context, it's
-challenging to consider "deep learning" as a new advance, though the term has
+challenging to consider "deep learning" as a new advance, `TODO: I'd argue the transition from multi-layer neural networks to deep learning is equivalent to the transition from theory to practice. The core concepts were there but architectures were discussed in the 80's, computationally we couldn't effectively train them until 00's this led to key breakthroughs in training, dropout, batch normalization, popularization of relu activation function. etc` though the term has
```
Comment (cgreene) [inline sections/02_intro.md]: Can you consider this change in the context of the most recent edits? It'd be nice to see a merge with these here. If you're citing specific techniques, you should reference the original papers on those advances to make it easy for a reader to follow.

File: sections/02_intro.md [hunk 6]
```diff
@@ -85,10 +81,7 @@ considered infeasible, or if it made difficult, tedious, and non-routine
 analyses routine.
 
 Based on our guiding question, we focused on the application of deep learning to
-topics of biomedical importance. This covers a large range of topics. We divided
-these into three broad classes based on their applied areas. We briefly
-introduce the types of questions, approaches and data which are typical for each
-class in the application of deep learning.
+topics of biomedical importance. We divided the large range of topics into three broad classes based on their applied areas: (1) Disease and Patient Categorization, (2) Fundamental Biological Study, and (3) Patient Treatment. We briefly introduce the types of questions, approaches and data which are typical for each class in the application of deep learning.
```
Comment (cgreene) [inline sections/02_intro.md]: I like listing the topics in text, but think we should drop the numbers since we won't be numbering the section headers.

File: sections/02_intro.md (hunk 2 above)
Comment (brettbj) [inline sections/02_intro.md]: ok, going with researcher, seems a more plausible first step

File: sections/02_intro.md (hunk 4 above)
Comment (brettbj) [inline sections/02_intro.md]: Exactly - http://www.nature.com/nature/journal/v521/n7553/full/nature14539.html

File: sections/02_intro.md (hunk 5 above)
Comment (brettbj) [inline sections/02_intro.md]: removed based on update for now, happy to write a "why now" portion if we think it's appropriate - NN's in 50's, backprop in 80's, restricted botlzmann, gpu's in 2000's, popularity of training techniques - relu, dropout, batch normalization >2010

File: sections/02_intro.md (hunk 4 above)
Comment (cgreene) [inline sections/02_intro.md]: In the TODO can you put the DOI of that article and specific panel so that others can easily see what you mean.

File: sections/02_intro.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
+also new challenges. We expect a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
```
Comment (cgreene) [inline sections/02_intro.md]: Can you use this language here? Right now your changes hit a sentence that @gwaygenomics also touched. I suspect your merge will wipe out his:
> We expect that achieving transformational advances will require algorithms that automatically extract meaningful patterns while providing actionable insights that enable us to better treat, study, and categorize disease, all within data constrained and privacy critical environments.

File: sections/02_intro.md [hunk 8]
```diff
@@ -16,12 +16,12 @@ demonstrated that a neural network could learn to identify cats simply by
 watching online videos [@doi:10.1109/ICASSP.2013.6639343]. Such approaches,
 termed deep learning, seem like a solution to the challenge presented by the
 growth of data in biomedicine. Perhaps these algorithms could identify the
-biological "cats" hidden in our data - the patterns unknown to researcher - and could act on them.
+biological "cats" hidden in our data - the patterns unknown to the researcher - and allow the researcher to act on them.
```
Comment (cgreene) [inline sections/02_intro.md]: Same here with biological. Please remove. Since it hits the same line, merging is going to result in this being re-added.

**Comment by cgreene** [comment]
It looks like you are still in the middle of updating to address change suggestions. Please note here when you're ready for another pass through this. Thanks!

**Comment by brettbj** [comment]
@cgreene - Addressed change suggestions and resolved citation formats to contribution guidelines

File: sections/02_intro.md (hunk 4 above)
Comment (agitter) [inline sections/02_intro.md]: I see no way to incorporate this into a static manuscript, but I really love http://playground.tensorflow.org/ for introducing non-linear classification with neural networks.  I'm not suggesting anything actionable with this comment, just leaving it on the small chance that we would want to make our own figure instead of reusing the one @brettbj suggested.
